app/knights.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Knight:
+    knight: str
```

Comment:
The `knight` attribute seems redundant as it duplicates the `name` attribute. Consider removing it to avoid confusion and redundancy.